modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase

```diff
@@ -14,6 +14,96 @@ definition {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+			var rangesArray = '''[{'range': '[0 TO 59.99]'}\, {'range': '[60 TO 99.99]'}\, {'range': '[100 TO 199.99]'}\, {'range': '[200 TO 499.99]'}\, {'range': '[500 TO 999.99]'}\, {'range': '[1000 TO *]'}]''';
+
+			CommerceWidget.updateFacetConfiguration(
+				portletName = "Price Range Facet",
+				rangesArray = ${rangesArray});
+		}
+
+		task ("When refresh the page") {
+			Refresh();
+		}
+
+		task ("Then assert the new ranges with the correct counts are shown") {
+			AssertElementPresent(
+				facetValueCount = 23,
+				facetValueName = "$ 0.00 - $ 59.99",
+				locator1 = "Search#FACET_DISPLAY_FREQUENCIES_COUNT");
+
+			AssertElementPresent(
+				facetValueCount = 10,
+				facetValueName = "$ 60.00 - $ 99.99",
+				locator1 = "Search#FACET_DISPLAY_FREQUENCIES_COUNT");
+
+			AssertElementPresent(
+				facetValueCount = 10,
+				facetValueName = "$ 100.00 - $ 199.99",
+				locator1 = "Search#FACET_DISPLAY_FREQUENCIES_COUNT");
+
+			AssertElementPresent(
+				facetValueCount = 4,
+				facetValueName = "$ 200.00 - $ 499.99",
+				locator1 = "Search#FACET_DISPLAY_FREQUENCIES_COUNT");
+
+			AssertElementPresent(
+				facetValueCount = 2,
+				facetValueName = "$ 500.00 - $ 999.99",
+				locator1 = "Search#FACET_DISPLAY_FREQUENCIES_COUNT");
+
+			AssertElementPresent(
+				facetValueCount = 0,
+				facetValueName = "$ 1,000.00+",
+				locator1 = "Search#FACET_DISPLAY_FREQUENCIES_COUNT");
+		}
+
+		task ("And then the user is able to filter products using the first two ranges on the newly set Price Range Facet widget") {
+			Check.checkToggleSwitch(
+				key_fieldName = "$ 0.00 - $ 59.99",
+				locator1 = "FormFields#CHECKBOX");
+
+			var products = "U-Joint,Transmission Cooler Line Assembly,Torque Converters,Wheel Seal - Front,Mount,Transmission Fluid,Wheel Bearing Grease,Tie Rod End Tool,Bushings,Lift Support,Muffler/Resonators,Exhaust Clamps,Valve Cover Grommets,Timing Chain,Timing Chain Tensioner,Seal,Piston,Master Cylinder,Brake Rotors";
+
+			for (var i : list ${products}) {
+				AssertElementPresent(
+					key_productName = ${i},
+					locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
+			}
+
+			Click(
+				key_pageNumber = 2,
+				locator1 = "CommerceNavigation#PAGINATION_ANY_PAGE");
+
+			for (var i : list "Premium Brake Pads,Brake Pads,ABS Sensor") {
+				AssertElementPresent(
+					key_productName = ${i},
+					locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
+			}
+
+			CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Catalog");
+
+			Check.checkToggleSwitch(
+				key_fieldName = "$ 60.00 - $ 99.99",
+				locator1 = "FormFields#CHECKBOX");
+
+			var products = "CV Axles,Shock Absorber,Oil Pump,Bearings,Master Cylinder,Calipers,Wear Sensors,Premium Brake Fluid,Brake Fluid";
+
+			for (var i : list ${products}) {
+				AssertElementPresent(
+					key_productName = ${i},
+					locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
+			}
```

Comment:
Can you rename the 'i' in these lines to more meaningful names? e.g. product or productName